CHANGELOG.md

```diff
@@ -1,5 +1,183 @@
+const itemsCount       = ReactSelector('TodoApp div span');
+```
+
+And it enables you to get React component's `state` and `props`.
+
+```js
+import ReactSelector from 'testcafe-react-selector';
+
+fixture `TODO list test`
+    .page('http://localhost:1337');
+
+test('Check list item', async t => {
+    const el = ReactSelector('TodoList');
+
+    await t.expect(el.getReact().props.priority).eql('High');
+    await t.expect(el.getReact().state.isActive).eql(false);
+});
+```
+
+To learn more, see the [testcafe-react-selectors](https://github.com/DevExpress/testcafe-react-selectors/) repository.
+
+#### :gear: Plugin for testing Vue.js apps
+
+In addition to the React plugin, we have released a plugin that facilitates testing Vue.js applications.
+
+In the same manner, it allows you to select Vue.js components with special `VueSelector` selectors.
+
+```js
+import VueSelector from 'testcafe-vue-selectors';
+
+const rootVue   = VueSelector();
+const todoInput = VueSelector('todo-input');
+const todoItem  = VueSelector('todo-list todo-item');
+```
+
+These selectors allow you to get Vue component's `props`, `state` and `computed` properties.
+
+```js
+import VueSelector from 'testcafe-vue-selector';
+
+fixture `TODO list test`
+    .page('http://localhost:1337');
+
+test('Check list item', async t => {
+    const todoItem = VueSelector('todo-item');
+
+    await t
+        .expect(todoItem.getVue().props.priority).eql('High')
+        .expect(todoItem.getVue().state.isActive).eql(false)
+        .expect(todoItem.getVue().computed.text).eql('Item 1');
+});
+```
+
+To learn more, see the [testcafe-vue-selectors](https://github.com/DevExpress/testcafe-vue-selectors) repository.
+
+#### :gear: TestCafe Docker image ([#1141](https://github.com/DevExpress/testcafe/issues/1141))
+
+We have created a Docker image with TestCafe, Chromium and Firefox preinstalled.
+
+You no longer need to manually install browsers or the testing framework on your server.
+Pull the Docker image from the repository and run TestCafe immediately.
+
+```sh
+docker pull testcafe/testcafe
+docker run -v //user/tests:/tests -it testcafe/testcafe firefox tests/**/*.js
+```
+
+To learn more, see [Using TestCafe Docker Image](https://devexpress.github.io/testcafe/documentation/using-testcafe/installing-testcafe.html#using-testcafe-docker-image)
+
+#### :gear: Support for Internet access proxies ([#1206](https://github.com/DevExpress/testcafe/issues/1206))
+
+If your local network uses a proxy server to access the Internet, TestCafe can use it reach the external webpages.
+
+To specify the proxy server, use a command line option
+
+```sh
+testcafe chrome my-tests/**/*.js --proxy 172.0.10.10:8080
+```
+
+or a method in the API.
+
+```js
+runner.useProxy('username:password@proxy.mycorp.com');
+```
+
+Note that you can pass the credentials with the proxy server host.
+
+#### :gear: Debugging mode option in CLI ([#1347](https://github.com/DevExpress/testcafe/issues/1347))
```

Comment:
We also need include information about option for Programming API: `runner.run({ debugMode: true })`.
And in [basic documentation topic](https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/using-testcafe/programming-interface/runner.md#run) too.